### PR TITLE
🐛 Fix 'make down'

### PR DESCRIPTION
**Pull request by wvangeit**
## What do these changes do?

Some changes to prevent the following three messages from occurring during the running of 'make down' in the root of the repo. (Especially if one runs make down several times consecutively) 

1)
`# Removing client containers (if any)
[+] Running 1/0
 ! Network tools_default  Resource is still in use `

2)
`# Removing generated docker compose configurations, i.e. .stack-*
rm: missing operand
Try 'rm --help' for more information.
make: [Makefile:396: down] Error 1 (ignored)`

3)
`# Removing local registry if any
Error response from daemon: No such container: registry`

## How to test

Run 'make down' after 'make up-devel', even several times in a row. No error messages should be generated.

## DevOps Checklist

N/A

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
Thanks! :1st_place_medal:

**Review by pcrespov — APPROVED** [review]
🎉 highly appreciated!

**Review by sanderegg — APPROVED** [review]
thank you very much for fixing these!
We do use long option names when available in scripts and Makefiles for better readability

File: Makefile [hunk 1]
```diff
@@ -395,9 +395,11 @@ down: ## Stops and removes stack
 	# Removing client containers (if any)
 	-@$(MAKE_C) services/static-webserver/client down
 	# Removing generated docker compose configurations, i.e. .stack-*
+ifneq ($(wildcard .stack-*), )
 	-@rm $(wildcard .stack-*)
+endif
 	# Removing local registry if any
-	-@docker rm --force $(LOCAL_REGISTRY_HOSTNAME)
+	-@docker ps -aq --filter "name=$(LOCAL_REGISTRY_HOSTNAME)" | xargs -r docker rm
```
[the remark this answers is not in the document]
Comment (wvangeit) [reply Makefile]: Makes sense, fixed: https://github.com/ITISFoundation/osparc-simcore/pull/4881/commits/72d224d6e1e8d2f77253bf0cb559252b37293c30#

**Review by odeimaiz — APPROVED** [review]
👍

**Pull request merged**